### ISSUE TITLE
Performance Toolkit Enhancements

### ISSUE THREAD

**Issue by Stoyvo** [magento/community-features#147]
Originally a fix for this item, was asked to move it to this repo.  https://github.com/magento/magento2/issues/21491

The performance toolkit will insert static data into the DB, however it's missed some critical items such as Invoices. When an invoice is created the order status will change to "Processing". Only after an order is set to Processing will the Life Time sales calculate on the Admin Dashboard.

If Magento had this information, they may have caught that Admin login can take upwards of 5-10 seconds just to calculate the life time sales (over 1 Million invoiced orders). Majority of admin users have a focus to perform an action (process an order, config change, flush cache, edit product, etc) and don't need the delay life time sales calculation as it's ignored. This is a separate issue to be reported. 

This update to performance toolkit provides us the ability to populate this data to create the issue.

A Pull Request has already been made and requires Magento review: https://github.com/magento/magento2/pull/21492

Cheers
